examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,94 @@
+#  Cross Validation for Binary Classifier
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_binary_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_binary_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_binary_2d_linear_labels_test.dat")
+Math:init_random(2)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+BinaryLabels labels_train(f_labels_train)
```

Comment:
use Labels and the new API

Reply:
Now uses Labels. Fixed in the coming commit.

Reply:
Tried changing to Labels but has problems with StratifiedCrossValidation (I opened an issue with details) so I'm leaving it BinaryLabels for now.

Reply:
As said in #4299 you will need to patch the .cpp file and convert the labels to the right type rather than asserting it.

Reply:
Did you attempt doing this, @FaroukY?